test/test_estimates_api.py

```diff
@@ -142,6 +142,18 @@ def test_create_bitcoin_estimate_transaction_value(self):
             estimate.data.mass_g, 200
         )  # not setting an exact value since this is changing daily
 
+    def test_create_ethereum_estimate_transaction_value(self):
```

Comment:
Should there be tests for `project_id`,`timestamp`, and `create_order` params as well?